src/utils/validations.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+  numValidation: (score: string, age: string) => {
+    return parseInt(score) < 11 &&
+      parseInt(score) >= 0 &&
+      parseInt(age) > 0 &&
+      parseInt(age) < 199
+      ? true
+      : false;
+  },
+  stringValidation: (str: string) => {
+    const reg = new RegExp("^([a-zA-Zà-úÀ-Ú])+$");
+    return reg.test(str) ? true : false;
+  },
```

Comment:
passar `new RegExp("^([a-zA-Zà-úÀ-Ú])+$")` para constante e setar uma nova toda vez que o método é chamado
```suggestion
  stringValidation: (value: string) => {
    return new RegExp("^([a-zA-Zà-úÀ-Ú])+$").test(value);
  },
```